### PR TITLE
Updates homepage of documentation

### PR DESCRIPTION
**Pull request by froschdesign**

### CODE REVIEW

File: docs/book/index.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+[Learn More](v3/getting-started/quick-start.md){: .btn .btn-lg .btn-primary}
+
+## Applications, Simplified
+
+Write middleware:
+
+```php
+$pathMiddleware = function (
+ServerRequestInterface $request,
```
Comment (michalbundyra) [inline docs/book/index.md]: missing indent for function arguments

File: mkdocs.yml [hunk 2]
```diff
@@ -251,7 +252,7 @@ pages:
             - '_migration_from_rc5_and_earlier': reference/migration/rc-to-v1.md
             - '_migration_to_expressive_1.1': reference/migration/to-v1-1.md
 site_name: Expressive
-site_description: 'zend-expressive: PSR-7 Middleware Microframework'
+site_description: 'PSR-15 Middleware in Minutes'
 repo_url: 'https://github.com/zendframework/zend-expressive'
 copyright: 'Copyright (c) 2015-2018 <a href="https://www.zend.com/">Zend Technologies USA Inc.</a>'
```
Comment (michalbundyra) [inline mkdocs.yml]: we can remove above line

**Review by michalbundyra — CHANGES_REQUESTED** [review]

File: docs/book/index.md (hunk 1 above)
Comment (froschdesign) [reply docs/book/index.md]: Good catch!

**Comment by michalbundyra** [comment]
Thanks, @froschdesign!

**Review by michalbundyra — APPROVED** [review]

**Pull request merged**